emission/storage/timeseries/builtin_timeseries.py

```diff
@@ -54,6 +54,7 @@ def __init__(self, user_id):
                 "stats/server_api_time": self.timeseries_db,
                 "stats/server_api_error": self.timeseries_db,
                 "stats/pipeline_time": self.timeseries_db,
+                "stats/dashboard_time": self.timeseries_db,
```

Comment:
this should be a `non_user_timeseries_db` instead of either `timeseries` or `analysis_timeseries`, both of which are user keyed.